src/test/java/com/hyperwallet/clientsdk/model/HyperwalletVenmoAccountTest.java

```diff
@@ -1,5 +1,7 @@
 package com.hyperwallet.clientsdk.model;
 
+import com.hyperwallet.clientsdk.model.HyperwalletVenmoAccount.Type;
```

Comment:
Unused import. Please remove all the unused imports in all the files.

Reply:
done